inst/lintr/linters.csv

```diff
@@ -57,6 +57,7 @@ namespace_linter,correctness robustness configurable executing
 nested_ifelse_linter,efficiency readability
 no_tab_linter,style consistency deprecated
 nonportable_path_linter,robustness best_practices configurable
+nrow_subset_linter,efficiency consistency readability best_practices
```

Comment:
I don't really see how this helps readability tbh.

Reply:
Fair... I think of it in a context where I often see things like `if (nrow(subset(x, many & con & ditions)) > 0) { x <- subset(x, many & con & ditions) }` where there's a readability issue but it's not strictly about the `nrow(subset(.))` call.

I removed the tag.